pkg/storaged/format-dialog.jsx

```diff
@@ -283,11 +282,6 @@ export function format_dialog(client, path, start, size) {
                                 { validate: (name, vals) => utils.validate_fsys_label(name, vals.type),
                                   visible: is_filesystem
                                 }),
-                      TextInput("custom", _("Filesystem type"),
-                                { visible: function (vals) {
-                                    return vals.type == "custom";
-                                }
-                                }),
```

Comment:
Seems we don't even have an integration test for that. :-) But what's the rationale here, just to unclutter the UI and leave people to the CLI if they know what they are doing?

Reply:
Yes, exactly.